spec/requests/cloud_volume_types_spec.rb

```diff
@@ -0,0 +1,44 @@
+#
+# REST API Request Tests - Cloud Volume Types
+#
+# Regions primary collections:
+#   /api/cloud_volume_types
+#
+# Tests for:
+# GET /api/cloud_volume_types/:id
+#
+
+describe "Cloud Volume Types API" do
+  it "forbids access to cloud volume types without an appropriate role" do
```

Comment:
Should these first two tests be wrapped in a context since they should be testing the collection?